docker.mk

```diff
@@ -82,6 +82,15 @@ drush:
 logs:
 	@docker-compose logs -f $(filter-out $@,$(MAKECMDGOALS))
 
+## debug	:	Enable xdebug.
+.PHONY: debug
+debug:
+	@echo "Starting up containers for $(PROJECT_NAME) with xdebug enabled."
+	@echo "¡¡CAUTION!! X-debug will only work if you have correctly configured docker-compose.xdebug.override.yml file."
+	@docker-compose stop
```

Comment:
Why docker-compose is stopped? Once you add the override file (line 92) the PHP container will be recreated without needing to previously stop it.

Reply:
You are right, php contaniner is recreated, but unfortunately docker-compose not include new override files if containers are already up. I tried this without stopping composer but xdebug don't get enabled.

Reply:
Then, it would be okay to only stop the PHP container?  It would work? We could do the same for pull / up commands, as Xdebug only affects PHP.